pyproject.toml

```diff
@@ -1,25 +1,27 @@
 [project]
-name="skema"
-authors=[
+name = "skema"
+authors = [
     {name="Enrique Noriega", email="enoriega@arizona.edu"},
```

Comment:
@myedibleenso We should add Keith here too

Reply:
@kwalcock , if you'd like to be added to the list of authors in this PR, please post your preferred email here.

Reply:
Thanks for thinking of me.  If it isn't specifically just for the Python code, which I haven't worked on, then it would be OK.  LUM would probably like to have it be keith@lum.ai.  If it is, then credit to where it is due, of course.

Reply:
@kwalcock is being too humble. Of course he should be listed